python-threatexchange/threatexchange/cli/match_cmd.py

```diff
@@ -215,6 +221,90 @@ def execute(self, settings: CLISettings) -> None:
                             fetched_data,
                         )
 
+    def validate(
```

Comment:
nit: This is still only called with `--hashes`, not in all calls will use it

Reply:
Cool, will update -- should we also be validating the non `--hashes` calls as well? I can update either in this pr or a follow up one.